.github/workflows/release-python-runtime.yml

```diff
@@ -27,9 +27,55 @@ jobs:
           R2_ACCOUNT_ID: ${{ secrets.PYODIDE_CAPNP_R2_ACCOUNT_ID }}
           R2_ACCESS_KEY_ID: ${{ secrets.PYODIDE_CAPNP_R2_ACCESS_KEY_ID }}
           R2_SECRET_ACCESS_KEY: ${{ secrets.PYODIDE_CAPNP_R2_SECRET_ACCESS_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           bazel build @workerd//src/pyodide:python_bundles @workerd//src/pyodide:bundle_version_info --remote_cache=https://bazel:${{ secrets.BAZEL_CACHE_KEY }}@bazel-remote-cache.devprod.cloudflare.dev
           # boto3 v1.36.0 fails with:
           # NotImplemented error occurred in CreateMultipartUpload operation: Header 'x-amz-checksum-algorithm' with value 'CRC32' not implemented
           pip install 'boto3<1.36.0' requests
           python3 src/pyodide/upload_bundles.py
+
+      - name: Check for open PR and commit changes
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Commit changes to python_metadata.bzl and push to branch
+          # Configure git
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+
+          # Get current branch name
+          BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+          echo "Current branch: $BRANCH_NAME"
+
+          # Check if there are changes to python_metadata.bzl
+          if git diff --quiet build/python_metadata.bzl; then
+            echo "No changes to python_metadata.bzl"
+            exit 0
+          fi
+
+          echo "Changes detected in python_metadata.bzl"
+
+          # Check if there's an open PR for this branch
+          PR_COUNT=$(gh pr list --head "$BRANCH_NAME" --state open --json number --jq length)
+
+          if [ "$PR_COUNT" -eq 0 ]; then
+            echo "No open PR found for branch $BRANCH_NAME, skipping commit"
+            exit 0
+          fi
+
+          echo "Found open PR for branch $BRANCH_NAME"
+
+          # Commit and push the changes
+          git add build/python_metadata.bzl
+          git commit -m "Update python_metadata.bzl with new bundle info
```

Comment:
Does this really work as a multi-line string here?

Reply:
Apparently yes:
https://github.com/cloudflare/workerd/pull/4838/commits/42a32b346ad845adc6149cca6eabacaf37f0fbd8
I was a bit surprised by this too...